.prettierignore

```diff
@@ -1,42 +1,47 @@
-# Ignore node_modules
+# Node modules
 node_modules/
 
-# Ignore build output
+# Build outputs
 dist/
 build/
 out/
+.next/
+.vercel/
 
-# Ignore coverage reports
+# Coverage
 coverage/
 
-# Ignore environment files
+# Environment files
 .env
 .env.*
 
-# Ignore lock files
+# Lock files
 package-lock.json
 yarn.lock
 pnpm-lock.yaml
 
-# Ignore logs
+# Logs
 *.log
 
-# Ignore misc
+# Misc
 .DS_Store
+tmp/
+temp/
 
-# Ignore public assets
+# Public assets
 public/
 
-# Ignore generated files
+# Generated files
 *.min.js
 *.map
 
-# Ignore next.js build output
-.next/
+# Markdown files (skip formatting)
+*.md
 
-# Ignore vercel build output
-.vercel/
+# Dependencies
+.pnp
+.pnp.js
```

Comment:
Hmm not familiar of `pnp` and `.pnp.js` are they part of the docs ignore rules as well?